extras/userdoc/md/documentation/index.md

```diff
@@ -153,17 +148,15 @@ we will add it to our [publication list](publications.md "Publications"),
 thus making it visible to potential readers. Send us your reference or even a
 reprint, using the mail address given on the [contact page](impressum.md "Impressum").
 
-NEST logo for your poster or presentation
------------------------------------------
+## NEST logo for your poster or presentation
 
 If you like NEST, why not show it on your poster or on your slides?
 
 <https://github.com/nest/nest-simulator/tree/master/extras/logos>
 
 ![](../../../logos/nest-simulated.png)
```

Comment:
This image is too large on the current homepage. Can we make it smaller, please?